docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+services:
+  application:
+    container_name: template-node-ts
+    build:
+      context: .
+      dockerfile: Dockerfile.development
+    command: sh -c "npm install && npm run start:dev"
+    volumes:
+      - .:/usr/src/app
```

Comment:
I don't think the volume is necessary because all development should be ephemeral, and not stateful, so we should only be able to spin up a container, do our work, and the next iteration of it should remove all from the previous run.

Besides, the application shouldn't keep local state, if this is necessary, it can be added ad hoc